runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillTimerInternals.java

```diff
@@ -285,13 +304,27 @@ public static TimerData windmillTimerToTimerData(
    * <p>This is necessary because Windmill will deduplicate based only on this tag.
    */
   public static ByteString timerTag(WindmillNamespacePrefix prefix, TimerData timerData) {
```

Comment:
the timer tag is parsed in windmillTimerToTimerData. We need to update that function to be able to parse a TimerData out of the timer tag, otherwise the timer family will always get lost.

Reply:
@reuvenlax , parsing is done.